### PR TITLE
bump `rsvp`

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Comment by rwjblue** [comment]
Is the bump to 4.0.2 deliberate this time? 😜

**Comment by bekzod** [comment]
yeah, I think well deserved bump :P

**Comment by Serabe** [comment]
Relaunched saucelabs

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "ember-router-generator": "^1.2.3",
     "jquery": "^3.2.1",
     "resolve": "^1.3.3",
-    "rsvp": "^3.6.1",
+    "rsvp": "^4.0.2",
```
Comment (rwjblue) [inline package.json]: Can you update this to 4.7.0 now that it is published?

**Review by rwjblue — COMMENTED** [review]
Can you also update the commit prefix to `[BUGFIX release]`?

File: package.json (hunk 1 above)
Comment (bekzod) [reply package.json]: done 👍

**Comment by rwjblue** [comment]
Awesome, thank you!

**Review by rwjblue — APPROVED** [review]

**Pull request merged**